core/templates/dev/head/css/oppia.css

```diff
@@ -3890,6 +3890,129 @@ md-card.preview-conversation-skin-supplemental-card {
   }
 }
 
+.aggregated-stats-supplemental {
+  padding-top: 30px;
+  margin-bottom: 30px;
+}
+
+.aggregated-stats-card {
+  margin: 0;
+}
+
+.stats-card-supplemental {
+  text-align: center;
+  padding: 0;
+}
+
+.option-card-supplemental {
+  padding: 10px; margin: auto
+}
+
+.option-card-text {
+  font-size: 14px;
+}
+
+.protractor-test-improvements-action-button {
+  margin-left: 15px;
+}
```

Comment:
Don't use classes that are prefixed by `.protractor-`, these are solely for testing purposes.